jib-core/src/main/java/com/google/cloud/tools/jib/image/json/V22ManifestListTemplate.java

```diff
@@ -92,13 +92,7 @@ public int getSchemaVersion() {
         .collect(Collectors.toList());
   }
 
-  /**
-   * Template for inner JSON object representing content descriptor for a layer or container
-   * configuration.
-   *
-   * @see <a href="https://github.com/opencontainers/image-spec/blob/master/descriptor.md">OCI
-   *     Content Descriptors</a>
-   */
+  /** Template for inner JSON object a single platform specific manifest. */
```

Comment:
Template for inner JSON object **representing** a single platform specific manifest.

Reply:
ugh :+1: